src/lib/common/github/issuereporter.ts

```diff
@@ -0,0 +1,223 @@
+import * as Octokit from '@octokit/rest';
+import { IssuesUpdateParams } from '@octokit/rest';
+
+import { IssueData } from '../../types/issuedata';
+
+const { NODE_ENV, environment } = process.env; // eslint-disable-line no-process-env
+
+const production = NODE_ENV !== 'development';
+
+// - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
+
+export class IssueReporter {
+
+    /* eslint-disable no-process-env */
+    private GITHUB_API_TOKEN = process.env.GITHUB_API_TOKEN;
+    private GITHUB_OWNER = process.env.GITHUB_OWNER;
+    private GITHUB_REPO = process.env.GITHUB_REPO;
+    private GITHUB_DATA;
+    /* eslint-enable no-process-env */
+
+    private octokit;
```

Comment:
Add type to `octokit`
```suggestion
    private octokit: Octokit;
```